tests/fixtures/TestService.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+ * (c) Lee Industrial Contracting
```

Comment:
I don't think we can accept a contribution with a separate copyright. Would you be OK with removing this doc block and having the file fall under the same license and copyright as the rest of the repository?

Reply:
lol holy crap. That's embarassing. Stupid PHPStorm. I'll take care of that right now.